app/default/src/ValueObject/PagesTotal.php

```diff
@@ -5,12 +5,14 @@
 
 final class PagesTotal
 {
-	private const MIN_PAGE = 1;
+	private const MIN_PAGE = 4;
```

Comment:
@212223 why? 🤔

Reply:
That should not be committed. It was changed only for a while due to this [issue](https://github.com/lrynek/painless-car-rental/issues/4).